src/components/Book.js

```diff
@@ -4,6 +4,8 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
```

Comment:
- [ ] I'll also suggest you remove the comment out code in line 2 of this file. :smile: :wave: